org-formation/800-redirects/_tasks.yaml

```diff
@@ -222,3 +222,21 @@ GenieBPCProdAppDnsForward:
     SourceHostedZoneId: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-sagebio-app-zone-HostedZoneId']
     # the value of the CNAME record
     TargetHostName: !CopyValue ['genie-bpc-shiny-prod-DockerFargateStack-LoadBalancerDNS', !Ref GenieProdAccount]
+# forward dca-dev.app.sagebionetworks.org to data_curator-infra ALB
+# https://github.com/Sage-Bionetworks/data_curator-infra
+
+SynapseDockerRegistryDevDnsForward:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.9/templates/R53/cname.yaml
+  StackName: !Sub '${resourcePrefix}-synapse-docker-registry-dev-cname'
+  StackDescription: Setup a CNAME for synapse-docker-registry dev ALB
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref SynapseDevAccount
+  Parameters:
+    # the name of the CNAME record
+    SourceHostName: "docker.dev.sagebase.org"
```

Comment:
is the idea to have a separate docker registry for synapse prod as well?

Reply:
Yes, that will come in a subsequent PR, once I've checked that dev' works.

Reply:
seems ok, but why not just setup one docker registry for the entire company?  or at least just one specifically for  Synapse?

Reply:
@zaro0508 We have a dev' Synapse stack with its own Docker registry, allowing us to follow the normal dev/prod deployment pattern.  The prod' deployment (into the Synapse prod' account) will follow.